src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -182,6 +199,8 @@
       {{ statesPaidClaimSettledClaimantDetails(claim, csrf) }}
     {% elif state.status === ClaimStatus.CLAIMANT_REJECTED_STATES_PAID %}
       {{ statesPaidInFullRejectedClaimantClaimDetails(claim) }}
+    {% elif state.status === ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_DEFENCE_NO_DQ %}
+      {{ claimRejectedFreeMediationNOWithoutDQsForClaimantClaimDetails(claim) }}
```

Comment:
I don't see `CLAIMANT_ACCEPTED_DEFENDANT_DEFENCE_NO_DQ` this scenario, please check if it's required.

Reply:
this is redundant. it would just be claim settled and would be covered under CLAIMANT_ACCEPTED_DEFENDANT_DEFENCE. I shall remove CLAIMANT_ACCEPTED_DEFENDANT_DEFENCE_NO_DQ